From ff82d4c89d6ca771dea06bbaa06ddf3fed760402 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <jacmet@sunsite.dk>
Date: Sat, 10 Nov 2012 18:58:48 +0100
Subject: [PATCH] fix build with uClibc

uClibc doesn't have/need libio.h, so don't include it from sockaddr.h

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 support/include/sockaddr.h |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/support/include/sockaddr.h b/support/include/sockaddr.h
index 9af2543..5eef2ec 100644
--- a/support/include/sockaddr.h
+++ b/support/include/sockaddr.h
@@ -20,7 +20,10 @@
 #ifndef NFS_UTILS_SOCKADDR_H
 #define NFS_UTILS_SOCKADDR_H
 
-#include <libio.h>
+/* uClibc doesn't have/need libio.h */
+#ifndef __UCLIBC__
+#include <libio.h> 
+#endif
 #include <stdbool.h>
 #include <sys/socket.h>
 #include <netinet/in.h>
-- 
1.7.9.5

